symengine/basic.h

```diff
@@ -158,6 +158,9 @@ class Basic : public EnableRCPFromThis<Basic> {
     //! Returns the list of arguments
     virtual vec_basic get_args() const = 0;
 
+    //! Returns the numerator/denominator form of the expression
+    virtual void as_numer_denom(const Ptr<RCP<const Basic>> &numer, const Ptr<RCP<const Basic>> &denom) const;
```

Comment:
@isuruf But doesn't calling the function like
`x->as_numer_denom(outArg(numer), outArg(denom))`, look much more intuitive than this
`as_numer_denom(x, outArg(numer), outArg(denom))`? What's the reason for not making it a member function? Just curious